contracts/Oracle/DSAuth.sol

```diff
@@ -1,44 +1,49 @@
-pragma solidity ^0.4.21;
+pragma solidity ^0.4.24;
 
 
 contract DSAuthority {
```

Comment:
Modifying external contracts( contracts from Maker) is not a good practice. Have you double checked that they don't have any new version? 
Especially with their migration to multi collateral, they probably have developed new price feed mechanism.

Reply:
Yes I agree.

The thing is we use this just for the test in local ganache, and we just want to remove the warnings, cause we have so many of them that we cannot pay attention to ours anymore.

I think we should depend directly from their projects, but we are out of capacity for this, given that is just for the tests and we are not altering any logic. I take note so we might do it in the future, also happy if u PR us the proposal :)